portfolio.html

```diff
@@ -0,0 +1,233 @@
+                <img
+                  src="images/website.jpg"
+                  alt="Inspiration has no Borders"
+                  width="370"
+                  height="294"
+                  class="portfolio-image"
+                />
+                <div class="portfolio-info">
+                  <h3 class="portfolio-title">Inspiration has no Borders</h3>
+                  <p class="portfolio-type">Веб-сайт</p>
+                </div>
+              </a>
+            </li>
+            <li class="portfolio-item">
+              <a href="#" class="portfolio-link">
+                <img
+                  src="images/design.jpg"
+                  alt="Видання Limited Edition"
+                  width="370"
+                  height="294"
+                  class="portfolio-image"
+                />
+                <div class="portfolio-info">
+                  <h3 class="portfolio-title">Видання Limited Edition</h3>
+                  <p class="portfolio-type">Дизайн</p>
+                </div>
+              </a>
+            </li>
+            <li class="portfolio-item">
+              <a href="#" class="portfolio-link">
+                <img
+                  src="images/ProjectLAB.jpg"
+                  alt="Проєкт LAB"
+                  width="370"
+                  height="294"
+                  class="portfolio-image"
+                />
+                <div class="portfolio-info">
+                  <h3 class="portfolio-title">Проєкт LAB</h3>
+                  <p class="portfolio-type">Маркетинг</p>
+                </div>
+              </a>
+            </li>
+            <li class="portfolio-item">
+              <a href="#" class="portfolio-link">
+                <img
+                  src="images/Business.jpg"
+                  alt="Growing Business"
+                  width="370"
+                  height="294"
+                  class="portfolio-image"
+                />
+                <div class="portfolio-info">
+                  <h3 class="portfolio-title">Growing Business</h3>
+                  <p class="portfolio-type">Додаток</p>
+                </div>
+              </a>
+            </li>
+          </ul>
+        </div>
+      </section>
+    </main>
+<footer class="footer">
+      <div class="container footer-content">
+        <a href="index.html" class="logo">
+          <span class="logo-blue">Web</span><span class="logo-white">Studio</span>
+        </a>
+         <address class="footer-address">
+          <ul class="address-list">
+            <li>
+              <a class="physical-address" href="https://www.google.com/maps/place/%D0%B1%D1%83%D0%BB%D1%8C%D0%B2%D0%B0%D1%80+%D0%9B%D0%B5%D1%81%D1%96+%D0%A3%D0%BA%D1%80%D0%B0%D1%97%D0%BD%D0%BA%D0%B8,+26,+%D0%9A%D0%B8%D1%97%D0%B2,+02000/@50.4265807,30.5383858,17z/data=!3m1!4b1!4m6!3m5!1s0x40d4cf0e033ecbe9:0x57a4dffefec77da0!8m2!3d50.4265807!4d30.5383858!16s%2Fg%2F1tctq5jv?entry=ttu">м. Київ, пр-т Лесі Українки, 26</a>
```

Comment:
"rel noopener rel noreferrer" are HTML attributes that can be used when adding links to external and internal pages to prevent security issues. They are simple to use, but provide an extra layer of protection against COR attacks and can also help users avoid links in their Browser to open.